config/environments/development.rb

```diff
@@ -97,3 +97,9 @@
   end
 
 end
+
+
+ActiveMerchant::Billing::Base.mode = :test
+
+  ::GATEWAY = ActiveMerchant::Billing::StripeGateway.new(
+    :login => ENV['SECRET_KEY'])
```

Comment:
It provides the Gateway login for the application. I will try to use a better alternative for this as well.